web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/broken/enterpriseBeanWithNonPassivatingConstructorFieldInDecorator/beans.xml

```diff
@@ -2,6 +2,6 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" version="1.1" bean-discovery-mode="all">
    <decorators>
-      <class>org.jboss.cdi.tck.tests.context.passivating.broken.enterpriseBeanWithNonPassivatingInitializerInDecorator.BrokenDecorator</class>
+      <class>org.jboss.cdi.tck.tests.full.context.passivating.broken.enterpriseBeanWithNonPassivatingConstructorFieldInDecorator.BrokenDecorator</class>
```

Comment:
Why updating `enterpriseBeanWithNonPassivatingInitializerInDecorator` to `enterpriseBeanWithNonPassivatingConstructorFieldInDecorator`